rts/lra/hotel-service/pom.xml

```diff
@@ -72,20 +71,45 @@
             <artifactId>narayana-lra</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.narayana.rts</groupId>
+            <artifactId>lra-client</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.narayana.rts</groupId>
+            <artifactId>lra-service-base</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.narayana.rts</groupId>
+            <artifactId>lra-proxy-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
 
         <!-- JAX-RS -->
         <dependency>
             <groupId>jakarta.ws.rs</groupId>
             <artifactId>jakarta.ws.rs-api</artifactId>
-            <scope>provided</scope>
+            <scope>compile</scope>
         </dependency>
 
         <!-- JAXRS 2 Client API implementation -->
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-client</artifactId>
-            <scope>test</scope>
+            <scope>compile</scope>
         </dependency>
+        
+        <dependency>
+    		<groupId>commons-logging</groupId>
+    		<artifactId>commons-logging</artifactId>
+    		<version>1.2</version>
+		</dependency>
```

Comment:
In case you missed it, I will mention that the formatting of the dependency imports in a few places seems a bit unexpected (as in the `</dependency>` in a number of places is aligned under the previous element). Hopefully if you were to look at https://github.com/jbosstm/quickstart/pull/446/files you would see what I mean. I won't block for it but if you do decide to address it I will be happy to re-review.

Reply:
Updated, thanks